src/styles/card.scss

```diff
@@ -0,0 +1,124 @@
+$grayColor: #616070;
+$blackColor: #060b35;
+$whiteColor: #fff;
+$whiteShade: #f3f3f3;
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: block;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $whiteShade;
+  border-radius: 5px;
+  background-color: $whiteColor;
+  padding: 32px 16px 16px;
+  transition: transform 0.3s;
+
+  &__img {
+    display: block;
+    margin: 0 3px 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__content {
+    &--title {
+      margin: 0;
+      color: $blackColor;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    &--prcode {
+      color: $grayColor;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      margin: 4px 0 0;
+    }
+  }
+
+  &__rating {
+    margin-top: 16px;
+    height: 16px;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &--review {
+      margin: 0;
+      color: $blackColor;
+      font-weight: 400;
+      font-size: 10px;
+      text-align: right;
+    }
+  }
+
+  &__price {
+    margin: 24px 0 16px;
+    display: flex;
+    justify-content: space-between;
+
+    &--title {
+      margin: 0;
+      color: $grayColor;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &--number {
+      margin: 0;
+      color: $blackColor;
+      font-size: 16px;
+      font-weight: 700;
+      text-align: right;
+      line-height: 18px;
+    }
+  }
+
+  &__submit {
+
+    &--button {
+      margin: 0;
+      display: block;
+      background-color: #00acdc;
+      width: 100%;
+      height: 40px;
+      line-height: 40px;
+      cursor: pointer;
+      text-decoration: none;
+      border-radius: 5px;
+      border: 0;
+      color: $whiteColor;
+      font-size: 14px;
+      font-weight: 700;
+
+      text-align: center;
+      text-transform: uppercase;
+      box-sizing: border-box;
+      transition: background 0.3s, color 0.3s;
+
+      &:hover {
+        background-color: $whiteColor;
+        border: 1px solid #00acdc;
+        color: #00acdc;
+      }
+    }
+  }
+
+  &:hover {
+    cursor: pointer;
```

Comment:
`cursor: pointer;` Should be be default, not only on hover. Move into `&--button` selector